src/components/GroupedTaskStatusBadge/index.tsx

```diff
@@ -91,8 +89,16 @@ const BadgeContainer = styled.div<BadgeColorProps>`
   align-items: center;
   line-height: normal;
   ${({ border }) => border && `border-color: ${border};`}
-  ${({ fill }) => fill && `background-color: ${fill};`}
+  ${({ fill, isActive }) =>
+    fill &&
+    `background-color: ${isActive === false ? hexToRGBA(fill, 0.2) : fill};`}
   ${({ text }) => text && `color: ${text};`}
+  ${({ isActive }) => isActive && `border-width: 2px;`}
+
+  :hover {
+    ${({ isActive, fill }) =>
+      `background-color: ${isActive === false ? fill : hexToRGBA(fill, 0.2)};`}
+  }
```

Comment:
[design suggestion] I think it might be clearer if we just lower the opacity of the entire badge rather than just the fill:

https://github.com/evergreen-ci/spruce/assets/47064971/6fc5485e-5dbe-4e42-af26-70a560ea9664